_posts/randy/2022-07-24-how-video-takes-all-day.md

```diff
@@ -0,0 +1,189 @@
+
+With the broadcast air of expectation slot to fill, it boils down to a matter of time to make something that matters. 
+
+**CREATIVE CHICKEN – WHAT COMES FIRST?** 
+
+Writing a reasonably cohesive technology article is one skill to aspire to, but creating a short representative video clip to represent meandering words is another challenge altogether. 
+
+Some subjects are hard to visualize – other creative treatments can hatch out of nowhere. A single picture can say a thousand words. Here we have 300 picture frames to work with - upward potential for 300,000 words communicated inside a 10 second video. 
+
+Which of these words stand out to tell the tale in short? It helps to be the writer and have familiarity with the subject matter, but there Creative Gods can be nudged along with a few words of recollection. Thy ask: 
+
+     What is the point of the story? 
+     What are the keywords to focus on? 
+     What visuals support the storyline? 
+     What motion images will attract attention? 
+     What style will appeal to the target audience? 
+
+Because these words can be written does not mean entertaining video magic will instantly ensue from the same fingertips. The clock may run out before a good idea is fashioned or the imagining is done. Sometimes one will make half-baked yolks and get egg on the face, but a taste of failure will make for a better cook. 
+
+**HOW TO BE CONTENT** 
+
+In Part 1 of this [10-Second-Video-Takes-All-Day story](https://devtails.xyz/@randy/10-second-video-takes-all-day{:target="_blank"}), a shot list of a dozen elements within an-car-entertainment system theme was compiled. As decider and follower of my own rules, I chose media acquisition items that would not require physical movement, gasoline sniffing or covid exposure. 
+
+ There exists a vast set of [video post-production](https://www.youtube.com/watch?v=bH2XBvV3x7Y){:target="_blank"}resources awaiting the persistent internet wanderer. The more photography, audio and editing experience you have, the more likely it is you have acquired a library of familiar content goodies and the knowledge of where to find more. This saves time big time. 
+
+You may have it all in your head but treasure can only be found if there is a map and you dig in the right place. It may only take a few minutes to find a sound effect that will do, but another certain element that needs to be perfect may take hours to source. 
+
+The following project task list is an example of how much time it actually might take for executing simple steps within a short video project. The minutes listed for each item are arbitrary, loose and concurrent with the creative process. 
+
+Some time to experiment, go down rabbit holes, and change direction away from abandoned concepts is accounted for. Know that the longer proportion of time taken in the primary review and development stage, the quicker discovery and creation tasks can be attended to. 
+
+**POST-PRODUCTION TO POST-ONLINE TIMELINE** 
```

Comment:
This is a neat timeline to see laid out.  It's often hard to comprehend, I might eventually do something similar for writing software as I think it helps a newb understand where they should be spending time.